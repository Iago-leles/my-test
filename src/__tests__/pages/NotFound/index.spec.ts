import { mount } from '@vue/test-utils'
import { describe, it, expect } from 'vitest'

import NotFoundPage from '@/pages/ErrorNotFound.vue'

describe('NotFoundPage.vue', () => {
  it('should match snapshot', () => {
    const wrapper = mount(NotFoundPage)

    expect(wrapper.html()).toMatchSnapshot()
  })
})
