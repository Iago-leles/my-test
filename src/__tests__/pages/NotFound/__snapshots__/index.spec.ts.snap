// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`NotFoundPage.vue > should match snapshot 1`] = `
"<div class="fullscreen bg-blue text-white text-center q-pa-md flex flex-center">
  <div>
    <div style="font-size: 30vh;">404</div>
    <div class="text-h2" style="opacity: 0.4;">error.not_found.title</div><button class="q-btn q-btn-item non-selectable no-outline q-btn--unelevated q-btn--rectangle bg-white text-blue q-btn--actionable q-focusable q-hoverable q-btn--no-uppercase q-mt-xl" tabindex="0" type="button"><span class="q-focus-helper"></span><span class="q-btn__content text-center col items-center q-anchor--skip justify-center row"><span class="block">error.not_found.button</span></span></button>
  </div>
</div>"
`;
